README.md

```diff
@@ -162,6 +162,12 @@ your server.
 - Twitter: [@twitterhandle](https://twitter.com)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/fatima-najafi-75424a23a/)
 
+👤 **Faisal**
+
```

Comment:
- [ OPTIONAL ] Great job working on the deployment but I think the link is broken, for now is not required to deploy it but if you want to add it kindly fix the broken link 💪🏻💯🤩